templates/a0-widget-setup-form.php

```diff
@@ -5,7 +5,6 @@
 $form_title = isset( $instance[ 'form_title' ] ) ? $instance[ 'form_title' ] : '';
 $social_big_buttons = isset( $instance[ 'social_big_buttons' ] ) ? $instance[ 'social_big_buttons' ] : '';
 $gravatar = isset( $instance[ 'gravatar' ] ) ? $instance[ 'gravatar' ] : '';
-$show_icon = isset( $instance[ 'show_icon' ] ) ? $instance[ 'show_icon' ] : '';
```

Comment:
Unused option

Reply:
could this be considered a breaking change?

Reply:
No, not used at all in the file or anywhere else.